pkg/provider/provider.go

```diff
@@ -131,6 +132,9 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData, version stri
 	sslmode := d.Get("sslmode").(string)
 	application_name := fmt.Sprintf("terraform-provider-materialize v%s", version)
 
+	// Set the host in the utils package so that the region can be extracted from it
+	utils.Host = host
```

Comment:
This solution is just temporary for the migration. host is would be removed from the provider entirely when cloud resources are added?

Reply:
Yep exactly. The idea is to not have to ask the user to specify it manually.